src/components/form/examples/row-layout-schema.ts

```diff
@@ -82,6 +100,15 @@ export const schema = {
                     type: 'boolean',
                     title: 'Receive email notifications',
                     lime: {
+                        help: {
+                            value: '**Why do we need this information?** This enables our sales representatives to contact you personally. We will keep all your personal information safe!',
+                            readMoreLink: {
+                                href: 'https://www.lime-technologies.com/en/av-saas/',
+                                title: 'This link opens in a new tab',
+                                target: '_blank',
+                                text: 'Our Terms & Privacy Policies',
+                            },
+                        },
                         component: {
                             name: 'limel-switch',
                         },
```

Comment:
It seems when a component is specified, the help does not appear at all. <img width="480" alt="image" src="https://github.com/Lundalogik/lime-elements/assets/35954987/4e7ed848-e010-4070-b4c5-56dc32536fa3">

Do you know why @civing ?